src/blog/2022/10/db-migraton-01.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Planned Database Migration
+subtitle: Moving to encrypted storage
+description: Details ppcoming maintence period
+date: 2022-10-13
+authors: ["ben-hardill"]
+---
+
+# Planned Database Migration
+
+As part of an on going security [review](https://flowforge.com/product/security/#data-at-rest) of the FlowForge Managed offering we discovered that the backend database was not using encrypted storage. In keeping with industry best practices we plan to migrate the database to a new instance using encrypted at rest storage.
+
+## Impact
+
+To do this cleanly we are going to need to schdule a short period of downtime for the core FlowForge application. During this downtime the following functions will not be available
```

Comment:
We really need a CI task to check for typos. I'll get on that.

```suggestion
To do this cleanly we are going to need to schedule a short period of downtime for the core FlowForge application. During this downtime the following functions will not be available
```

Reply:
As this list isn't exhaustive I'd flip it around actually. 

What about:

Customers' Node-RED instance will remain running, though any features that depend on FlowForge will not operate as expected. This includes user sessions, project and team management, as well as the project nodes for inter-project communication.

Reply:
There's a proposal In https://github.com/flowforge/handbook/issues/129 @ZJvandeWeg